content/blog/2018-12-28-subquery.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Subquery Support
+created_at: 2018-12-28
+kind: article
+author_name: Ganesh Vernekar
+---
+
+## Introduction
+
+As the title suggests, a subquery is a part of a query, and allows you to do a range query within a query, which was not possible before. It has been a long-standing feature request [prometheus/prometheus/1227](https://github.com/prometheus/prometheus/issues/1227).
+
+The pull request for subquery support was recently merged into Prometheus [prometheus/prometheus/4831](https://github.com/prometheus/prometheus/pull/4831) and will be available in Prometheus 2.7. Let’s learn more about it below.
+
+## Motivation
+
+Previously if you wanted to have a range selection on a query for your alerting rules or graphing, it would require you to have a recording rule based on that query, and perform range selection on the metrics created by the recording rules.
+
+When you want some quick results on data spanning days or weeks, it can be quite a bit of wait until you have enough data in your recording rules before it can be used. Forgetting to add recording rules can be frustrating. And it would be tedious to create a recording rule for each step of a query.
+
+With subquery support, all the wait and frustration is taken care of.
+
+## Subquery
```

Comment:
This is still the singular, which is odd to use as a title in English. I'd either write "Subqueries" or "Subquery Support".